pkg/doc/verifiable/credential_ldp_test.go

```diff
@@ -204,46 +204,40 @@ func TestNewCredentialFromLinkedDataProof_JSONLD_Validation(t *testing.T) {
 }
 `
 
-		vcWithLdp, _, err := NewCredential([]byte(vcJSON), vcOptions...)
+		vcWithLdp, _, err := newTestCredential([]byte(vcJSON), vcOptions...)
 		r.Error(err)
 		r.EqualError(err, "JSON-LD doc has different structure after compaction")
 		r.Nil(vcWithLdp)
 	})
 
 	t.Run("VC with different mapped field", func(t *testing.T) {
+		localJSONLDContext := `
+{
+  "@context":
```

Comment:
I assume this is still inline due to being tiny?

Reply:
Yeah, this is tiny enough file to leave it inlined.